src/full/Agda/Interaction/Options/Warnings.hs

```diff
@@ -306,9 +311,12 @@ warningNameDescription = \case
   EmptyRewritePragma_              -> "Empty `REWRITE' pragmas."
   EmptyWhere_                      -> "Empty `where' blocks."
   InvalidCatchallPragma_           -> "`CATCHALL' pragmas before a non-function clause."
+  InvalidConstructor_              -> "`constructor' blocks may only contain type signatures for constructors."
+  InvalidConstructorBlock_         -> "No `constructor' blocks outside of `infix mutual' blocks."
```

Comment:
`infix mutual` is only a placeholder for a keyword determined by bikeshedding, I suppose?